src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -2783,7 +2783,13 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Make_class_abstract" xml:space="preserve">
     <value>Make class 'abstract'</value>
   </data>
+  <data name="Inline_0" xml:space="preserve">
+    <value>Inline '{0}'</value>
+  </data>
   <data name="Remove_async_modifier" xml:space="preserve">
     <value>Remove 'async' modifier</value>
   </data>
+  <data name="Inline_0_and_keep_1" xml:space="preserve">
+    <value>Inline '{0}' and keep '{1}'</value>
```

Comment:
how does `{0}` differ from `{1}` here?  Can you give an example of the text a user would see?

Reply:
They are the same. So you mean the name should be 'Inlined_0"? 
It's just for generate action like
'Inline Bar() and Keep Bar()'
'Inline Bar()' // This one remove the 'Bar'
Becuase the code action could be inlined so each of them should be meaningful.
Or we could have 
'Inline Bar()' // This one keeps 'Bar()'
'Inline Bar() and remove 'Bar()'' // This one remove the 'Bar

Reply:
i would have:

1. `Inline 'Foo'` or `Inline and remove 'Foo'`.  (i prefer the first.)
2. `Inline and keep 'Foo'`